integration-libs/opf/checkout/components/opf-checkout-billing-address-form/opf-checkout-billing-address-form.component.ts

```diff
@@ -36,8 +40,15 @@ export class OpfCheckoutBillingAddressFormComponent implements OnInit {
   countries$: Observable<Country[]>;
 
   ngOnInit() {
+    this.activeCartFacade.getActive().subscribe((cart) => (this.cart = cart));
     this.countries$ = this.service.getCountries();
+    this.userAddressService.loadAddresses();
+    this.service.setDefaultBillingAddress();
     this.service.getAddresses();
+    this.service.pickupNoDefaultAddress$.subscribe(() => {
```

Comment:
Could you please verify if this subscription is properly unsubscribed during the flow?